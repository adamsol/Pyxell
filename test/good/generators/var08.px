
func spread(a) def
    yield ...a
    print '-'

g = spread("qwertyui")
a: Char

for a, i, b, c in g, 0..., g, g do
    print i, b, a, c

print a
print {...g}
