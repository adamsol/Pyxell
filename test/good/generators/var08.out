0 w q e
1 t r y
-
r
{}
